seonshine_frontend/src/pages/orderManagement/components/OrderListTab/index.tsx

```diff
@@ -61,24 +61,49 @@ const OrderListTab = ({ orderDate }: OrderListTabPropsType) => {
   const columns = viewMode === 'summary' ? OrderListRestaurantTableHeader : OrderListTableHeader;
 
   const data: OrderListType[] = useMemo(() => {
-    if (viewMode === 'summary' && orderListSummary) {
-      return orderListSummary.data.map((order) => ({
-        ordered_items: order.item_name,
-        amount: order.count,
-      }));
-    }
-
-    if (viewMode === 'detail' && orderListDetail) {
-      return orderListDetail.data.map((order) => ({
-        restaurant_name: order.restaurant_name,
-        employee_name: order.username,
-        ordered_items: order.item_name,
-        date: order.submitted_time,
-      }));
-    }
-
-    return [];
-  }, [viewMode, orderListSummary, orderListDetail]);
+    const getOrderSummaryData = () => {
```

Comment:
with define data is array with useMemo, we shouldn't define function inside useMemo